test/unit-test/test_app_btc_multisig.c

```diff
@@ -158,9 +160,35 @@ static testcase_t _tests[] = {
                 2,
             },
     },
+
+    /** P2SH **/
+    {
+        .coin = BTCCoin_BTC,
+        .script_type = BTCScriptConfig_Multisig_ScriptType_P2WSH_P2SH,
+        .threshold = 2,
+        .xpubs_count = 2,
+        .xpubs =
+            {
+                "xpub6FEZ9Bv73h1vnE4TJG4QFj2RPXJhhsPbnXgFyH3ErLvpcZrDcynY65bhWga8PazWHLSLi23PoBhGcL"
+                "cYW6JRiJ12zXZ9Aop4LbAqsS3gtcy",
+                "xpub6EGAio99SxrurYgGH5BEzSiM4ZNedDX68RTGTSzGt5gk4STbs8B35ASC3RMdysGhJ7dJfffQcQEzFA"
```

Comment:
they are all equivalent, the version bytes (which determine the first letters) are stripped off after parsing. i use xpubs basically because the wally xpub parser only accepts `xpub` (in `btc_util_parse_xpub()`).